requirements.txt

```diff
@@ -20,3 +20,4 @@ websockets
 typing-extensions
 MDAnalysis
 nglview
+moviepy
```

Comment:
Is this really needed? You can do the same with Pillow or imageio, which we have as a dependency already...